DateTimeParseException when trying to launch container

When Testcontainers is refreshing its local image cache, it is using the command "inspect image" and parses the timestamp of the attribute "created". If this timestamp contains a time zone offset, then the following exception is thrown and Testcontainers fails to launch the container.
```
org.testcontainers.containers.ContainerLaunchException: Container startup failed
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=docker-registry.example.org/some/image:1.0, imagePullPolicy=DefaultPullPolicy())
Caused by: java.time.format.DateTimeParseException: Text '2020-07-24T07:17:51.365190246+02:00' could not be parsed at index 29
```
This happend with Docker version 18.06.3 on Linux.

--- a/core/src/main/java/org/testcontainers/images/ImageData.java
+++ b/core/src/main/java/org/testcontainers/images/ImageData.java
@@ -7,6 +7,7 @@ import lombok.NonNull;
 import lombok.Value;
 
 import java.time.Instant;
+import java.time.ZonedDateTime;
 
 @Value
 @Builder
@@ -17,7 +18,7 @@ public class ImageData {
 
     static ImageData from(InspectImageResponse inspectImageResponse) {
         return ImageData.builder()
-            .createdAt(Instant.parse(inspectImageResponse.getCreated()))
+            .createdAt(ZonedDateTime.parse(inspectImageResponse.getCreated()).toInstant())
             .build();
     }
 

--- a/core/src/test/java/org/testcontainers/images/ImageDataTest.java
+++ b/core/src/test/java/org/testcontainers/images/ImageDataTest.java
@@ -1,0 +1,27 @@
+package org.testcontainers.images;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+
+import org.junit.Test;
+
+import com.github.dockerjava.api.command.InspectImageResponse;
+
+public class ImageDataTest {
+
+    @Test
+    public void shouldReadTimestampWithoutOffsetFromInspectImageResponse() {
+        final String timestamp = "2020-07-27T18:23:31.365190246Z";
+        final ImageData imageData = ImageData.from(new InspectImageResponse().withCreated(timestamp));
+        assertThat(imageData.getCreatedAt()).isEqualTo(Instant.parse(timestamp));
+    }
+
+    @Test
+    public void shouldReadTimestampWithOffsetFromInspectImageResponse() {
+        final String timestamp = "2020-07-27T18:23:31.365190246+02:00";
+        final ImageData imageData = ImageData.from(new InspectImageResponse().withCreated(timestamp));
+        assertThat(imageData.getCreatedAt()).isEqualTo(Instant.parse("2020-07-27T16:23:31.365190246Z"));
+    }
+
+}

